source/.ebextensions/node-perf-cleanup.config

```diff
@@ -1,3 +1,8 @@
-commands:
-  cleanup_node_perf_files:
-    command: rm -rf /tmp/perf-*.map
\ No newline at end of file
+files:
+  "/opt/elasticbeanstalk/hooks/appdeploy/pre/47node_perf_cleanup.sh":
```

Comment:
I'm not sure if you chose 47 to go after https://github.com/mixmaxhq/eb-fix-npm/blob/master/source/.ebextensions/fix-npm.config#L53, but if so, I wouldn't worry too much about trying to use unique numbers—EB considers the entire filename. The important thing is that you alphabetize correctly with respect to AWS' _own_ hooks: https://github.com/mixmaxhq/eb-fix-npm/blob/master/source/.ebextensions/fix-npm.config#L7.